gaitmap_mad/gaitmap_mad/event_detection/_rampp_event_detection.py

```diff
@@ -153,6 +157,7 @@ class RamppEventDetection(_EventDetectionMixin, BaseEventDetection):
 
     ic_search_region_ms: Tuple[float, float]
```

Comment:
What happens to this value, if the stride-type is ic? This seems to be ignored and you use other hardcoded thresholds.